Orange/data/_contingency.pyx

```diff
@@ -1,54 +0,0 @@
-#cython: embedsignature=True
```

Comment:
Can there be a nice public interface like `Table.contingency(col or cols, target_col)` or `Table.contingency()` that infers from domain?